deploy-tools.gemspec

```diff
@@ -1,13 +1,17 @@
 Gem::Specification.new do |s|
   s.name        = 'deploy-tools'
-  s.version     = '0.1.4'
-  s.date        = '2022-06-28'
+  s.version     = '0.2.0'
```

Comment:
I'd gripe style-guide about the mix of apostrophes and quotes but if rubocop isn't complaining...

Reply:
The repo does have a rubocop config, but my VSCode wasn't automatically running it.
I checked rubocop against the gemspec file and will correct all offenses (plenty are single quote offenses).

Reply:
Rubocop also suggests to remove `date`. I will also do this.
> Gemspec/DeprecatedAttributeAssignment: Do not set `date` in gemspec.

It also suggested I must add the `required_ruby_version`. I will match the version in rubocop.yml => 2.7.

Finally, I am going to skip addressing `Gemspec/RequireMFA: `metadata['rubygems_mfa_required']` must be set to `'true'`.` for right now.

Reply:
I added rubocop via bundler - doing so allowed my VSCode to properly run the in-IDE rubocop checks. It also standardizes the dependency, so hope that addition is welcome.